assets/javascripts/discourse/templates/components/chat-composer-uploads.hbs

```diff
@@ -0,0 +1,19 @@
+{{#if showUploadsContainer}}
+  <div class="chat-composer-uploads-wrapper">
+    {{#each uploads as |upload|}}
+      {{chat-composer-upload
+        upload=upload
+        isDone=true
+        onCancel=(action "removeUpload" upload)
+      }}
+    {{/each}}
+    {{#each inProgressUploads as |upload|}}
+      {{chat-composer-upload
+        upload=upload
+        onCancel=(action "cancelUploading" upload)
+      }}
+    {{/each}}
+  </div>
+{{/if}}
+
+<input type="file" id={{fileUploadElementId}} accept={{acceptedFormats}} multiple>
```

Comment:
We have this exact same thing in core but we also check for `acceptsAllFormats`, which is in the JS code for chat uploads but never used in the template. Here is core:

https://github.com/discourse/discourse/blob/a0eddd0518ccd619a0a7661431cbaa7d6a93a98e/app/assets/javascripts/discourse/app/templates/components/composer-editor.hbs#L26-L30

Perhaps we could make a really tiny component for upload file input, that includes the `acceptedFormats` and `acceptsAllFormats` functions?  `{{upload-file-input id={{fileUploadElementId}} }}`